app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -1447,30 +1447,34 @@ object TelemetryWrapper {
     }
 
     @TelemetryDoc(
-            name = "Click Rate or Feedback",
+            name = "click Rate App",
             action = Category.ACTION,
             method = Method.CLICK,
             `object` = Object.FEEDBACK,
             value = "null,dismiss,positive,negative",
             extras = [TelemetryExtra(name = Extra.SOURCE, value = "contextual_hints,settings,notification")
             ])
     @JvmStatic
-    fun clickRateOrFeedback(value: String, source: String?) {
-        EventBuilder(Category.ACTION, Method.CLICK, Object.FEEDBACK, value)
+    fun clickRateApp(value: String?, source: String) {
+        val builder = EventBuilder(Category.ACTION, Method.CLICK, Object.FEEDBACK, value)
                 .extra(Extra.SOURCE, source)
-                .queue()
+
+        if (value == null) {    // when clicking notification event body, value is null
```

Comment:
Why do we only add version when value is null? This is weird since RATE_APP_NOTIFICATION_TELEMETRY_VERSION is shared here.

Reply:
I know this is logically correct but I slightly prefer we add the version for all telemetries and always send them? (RATE_APP_NOTIFICATION_TELEMETRY_VERSION = 3)